### PR TITLE
Use URL rather than image ID

### PR DESCRIPTION
**Pull request by will-moore**
Allows using various s3 URLs, instead of hard-coding it.

### CODE REVIEW

File: napari_idr.py [hunk 1]
```diff
@@ -45,55 +45,37 @@ def napari_get_reader(path: PathLike) -> Optional[ReaderFunction]:
...
+        raise e
 
     pyramid = []
     for resolution in resolutions:
-        root = 'idr/zarr/v0.1/%s.zarr/%s/' % (image_id, resolution)
-        store = s3fs.S3Map(root=root, s3=s3, check=False)
-        cached_store = zarr.LRUStoreCache(store, max_size=(2048 * 2**20))
         # data.shape is (t, c, z, y, x) by convention
-        data = da.from_zarr(cached_store)
+        data = da.from_zarr(f"{zarr_path}{resolution}")
```
Comment (will-moore) [inline napari_idr.py:78]: Do we lose any caching functionality here?
I haven't tested to *know* what difference the cache makes? cc @manics

Comment (manics) [reply napari_idr.py:78]: Depends how Napari behaves. Best way to check is to enable debug logging for S3fs and see whether repeat requests are made. It definitely made a difference with the initial prototype several weeks ago

Comment (will-moore) [reply napari_idr.py:78]: Just trying this on https://github.com/ome/ome-zarr-py and it the loss of any s3 logging is another issue with using this shortcut.

Comment (manics) [reply napari_idr.py:78]: It must be using a HTTP reader since it looks like a plain URL, there's nothing to indicate it's an S3 store.

**Pull request merged**